app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt

```diff
@@ -206,6 +208,26 @@ class MainActivity : BaseActivity() {
     }
 
     private fun setupBottomNavigation() = with(binding) {
+        bottomNavigation.findViewById<View>(R.id.menuFragment).setOnLongClickListener {
+            navController.navigate(R.id.switchUserActivity)
+            true
+        }
+
+        bottomNavigation.findViewById<View>(R.id.menuFragment).setOnClickListener {
+            navController.navigate(R.id.menuFragment)
+
+            if (it.tag != null && Date().time - (it.tag as Date).time < DOUBLE_TAP_DELAY) {
+                CoroutineScope(Dispatchers.IO).launch {
+                    trackAccountEvent("switch")
+                    AccountUtils.switchToNextUser()
+                }.also { navController.navigate(R.id.homeFragment) }
+
+                it.tag = null
+            } else {
+                it.tag = Date()
+            }
```

Comment:
We should probably also not switch the user on double tap if there's only a single user